content/tutorials/manage-the-rippled-server/configuration/run-rippled-in-reporting-mode.md

```diff
@@ -0,0 +1,375 @@
+---
+html: run-rippled-in-reporting-mode.html
+parent: configure-rippled.html
+blurb: Run a special operating mode of rippled that handles remote procedure calls (RPC) for validated data.
+labels:
+  - Core Server
+  - Blockchain
+top_nav_grouping: Popular Pages
+---
+# Run rippled in Reporting Mode
+
+[Reporting mode](rippled-server-modes.html) is a mode of the XRP Ledger core server specialized for serving [HTTP and WebSocket APIs](rippled-api.html).
+
+In reporting mode, the server does not connect to the peer-to-peer network. Instead, it uses gRPC to get validated data from one or more trusted servers that are connected to the P2P network. 
+
+It can then efficiently handle RPCs, reducing the load on `rippled` servers running in P2P mode.
+
+{{ include_svg("img/reporting-mode-basic-architecture.svg", "Figure 1: Working of `rippled` in reporting mode") }}
+
+The reporting mode of `rippled` uses two datastores:
+
+* The primary persistent datastore for `rippled` that includes transaction metadata, account states, and ledger headers. You can use NuDB or Cassandra as the primary persistent datastore.
+
+* PostgreSQL database to hold relational data, which is used mainly by [tx method][] and [account_tx method][]. 
+
+When a reporting mode server receives an API request, it loads the data from these data stores if possible. For requests that require data from the P2P network, the reporting mode forwards the request to a P2P server, and then passes the response back to the client.
+
+Multiple reporting mode servers can share access to the same network accessible databases (PostgreSQL and Cassandra); at any given time, only one reporting mode server writes to the databases, while all the others read from the databases. 
+
+## How to Run Reporting Mode
+
+### Prerequisites
+
+1. Ensure that your system meets the [system requirements](system-requirements.html).
+
+    **Note:** If you choose to use Cassandra as the database, the disk requirements for `rippled` will be lower as the data will not be stored on your local disk.  
+
+2. In order to run reporting mode, you also need to run one or more `rippled` servers in [P2P mode](install-rippled.html). Ensure that you have at least one `rippled` server running in P2P mode. 
+
+3. Install and configure the datastores required to run `rippled` in reporting mode. 
+    
+    1. Install Postgres. 
+
+        <!-- MULTICODE_BLOCK_START -->
+
+        *Linux*
+
+        1. Download and install PostgreSQL on Linux. For instructions, refer to https://www.postgresqltutorial.com/install-postgresql-linux/.
+        
+        2. Connect to the PostgreSQL Database Server using psql, and create a user `newuser` and a database `reporting`.
+
+        psql postgres
+	        CREATE ROLE newuser WITH LOGIN PASSWORD ‘password’;
+            ALTER ROLE newuser CREATEDB;
+        \q
+        psql postgres -U newuser
+        postgres=# create database reporting;
```

Comment:
To get this to display as a code block inside a bulleted list, you need specific criteria:

- You need a blank line above. (You're good here.)
- To make it display as a child of the most recent list item, you need to indent 4 spaces more than the innermost list item.
- Then you need to add another 4 spaces to make it a code block. Otherwise it'll get treated as regular paragraph text.

For these lines (ignoring the multicode tabs for a moment), you're 3 numbered lists deep, which means you need to indent a total of (4*3) + 4 + 4 = 20 spaces for this code block.

I find that when you're nesting that deep, it's a sign you should divide up the document into multiple sections, each with their own header, instead.